src/main/utils/PlaybackAPI.js

```diff
@@ -35,6 +42,10 @@ class PlaybackAPI {
         album: null,
         albumArt: null,
       },
+      rating: {
+        liked: false,
```

Comment:
I do not have this experiment maybe it has been removed?
According to this entry in the product forum it's gone for good: https://productforums.google.com/forum/#!topic/play/HwKmVF5N5YA